inference-engine/src/vpu/common/include/vpu/ngraph/operations/static_shape_non_maximum_suppression.hpp

```diff
@@ -5,33 +5,43 @@
+#include <ngraph/op/non_max_suppression.hpp>
 
 #include <memory>
 #include <vector>
 
 namespace ngraph { namespace vpu { namespace op {
 
-class StaticShapeNonMaxSuppression : public ngraph::op::v4::NonMaxSuppression {
+class StaticShapeNonMaxSuppression : public ngraph::op::v5::NonMaxSuppression {
 public:
     static constexpr NodeTypeInfo type_info{"StaticShapeNonMaxSuppression", 0};
     const NodeTypeInfo& get_type_info() const override { return type_info; }
     StaticShapeNonMaxSuppression() = default;
 
     StaticShapeNonMaxSuppression(const Output<Node>& boxes,
-                      const Output<Node>& scores,
-                      const Output<Node>& max_output_boxes_per_class,
-                      const Output<Node>& iou_threshold,
-                      const Output<Node>& score_threshold,
-                      const BoxEncodingType box_encoding = BoxEncodingType::CORNER,
-                      const bool sort_result_descending = true,
-                      const ngraph::element::Type& output_type = ngraph::element::i64);
+                                 const Output<Node>& scores,
+                                 const BoxEncodingType box_encoding = BoxEncodingType::CORNER,
+                                 const bool sort_result_descending = true,
+                                 const ngraph::element::Type& output_type = ngraph::element::i64);
 
     StaticShapeNonMaxSuppression(const Output<Node>& boxes,
-                      const Output<Node>& scores,
-                      const BoxEncodingType box_encoding = BoxEncodingType::CORNER,
-                      const bool sort_result_descending = true,
-                      const ngraph::element::Type& output_type = ngraph::element::i64);
+                                 const Output<Node>& scores,
+                                 const Output<Node>& max_output_boxes_per_class,
+                                 const Output<Node>& iou_threshold,
+                                 const Output<Node>& score_threshold,
+                                 const BoxEncodingType box_encoding = BoxEncodingType::CORNER,
+                                 const bool sort_result_descending = true,
+                                 const ngraph::element::Type& output_type = ngraph::element::i64);
+
+    StaticShapeNonMaxSuppression(const Output<Node>& boxes,
+                                 const Output<Node>& scores,
+                                 const Output<Node>& max_output_boxes_per_class,
+                                 const Output<Node>& iou_threshold,
+                                 const Output<Node>& score_threshold,
+                                 const Output<Node>& soft_nms_sigma,
+                                 const BoxEncodingType box_encoding = BoxEncodingType::CORNER,
+                                 const bool sort_result_descending = true,
+                                 const ngraph::element::Type& output_type = ngraph::element::i64);
```

Comment:
Don't see where we need these constructors

Reply:
Actually one of them were used in DTS and the other were used in tests, but I have updated tests to use `softNMSSigma` parameter, and now I can just leave one constructor.

Reply:
Why constructor of base class is not enough?

Reply:
Do you mean I need to delete all the constructors from `StaticShapeNonMaxSuppression`?

Reply:
I mean, that if there is a reason for constructors in `StaticShapeNonMaxSuppression` - I'm ok for this, I just had an impression derived class is going to be just a wrapper that will override `validate_and_infer_types`

Reply:
This constructor just forwards its arguments to the base class, without it I will not be able to create `StaticShapeNMS` object with these arguments.